1001-2000/LIT1298Dersan.xml

```diff
@@ -7,6 +7,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <title xml:lang="gez" xml:id="t1">ድርሳነ፡ ሰንበት፡</title>
             <title xml:lang="gez" corresp="#t1" type="normalized">Dǝrsāna sanbat</title>
             <title xml:lang="en" corresp="#t1">Homily on the observance of Sunday</title>
+            
+            <title xml:lang="gez" xml:id="t2">ድርሳን፡ ለሰንበተ፡ ክርስቲያን፡</title>
```

Comment:
Fidal and transliteration don't correspond, but if they are both in use, we could add third title?